spec/plans/provision.fmf

```diff
@@ -155,7 +155,12 @@ example: |
 
         # Disk group used to allow possible future extensions
         disk:
-            space: 500 GB
+          - size: 500 GB
+
+        # Multiple disks can be requested as well
+        disk:
+          - size: 2 GB
+          - size: 20 GB
```

Comment:
```suggestion
          - size: >= 2 GB
          - size: >= 20 GB
```
Use an operator at least in some example to emphasize it is possible. More over, it is most likely required for such things as without it is it `=` I guess which may not be available due to various rounding approaches, at all.

Reply:
Good point, updated the `disk` section with `>=` examples.